starter_code/views/movieDetails.hbs

```diff
@@ -0,0 +1,23 @@
+<div class="movie-details-container">
+    <div>
+        <img class="movie-image" src=" {{this.image}} " alt="image of {{title}}">
+    </div>
+    <div class="movie-info">
+        <div>
+            <a href="/movies/">Go back</a>
+        </div>
+
+        <div>
+            <h2>{{title}}</h2>
+            <h3>Director: {{this.director}}</h3>
```

Comment:
Im pretty sure this.director == director, the same way as title == this.title.

It works and is not an issue, just wanted to let you know:)